src/tests/unit/auto/auto_default_perf_hint_test.cpp

```diff
@@ -216,12 +216,6 @@ class AutoDefaultPerfHintTest : public ::testing::TestWithParam<ConfigParams> {
             "AUTO",
             {"GPU"},
             {{"DEVICE_PROPERTIES", "GPU:{ALLOW_AUTO_BATCHING:FALSE}}"}, {"MULTI_DEVICE_PRIORITIES", "GPU"}}});  // GPU: no perf_hint
-
-        testConfigs.push_back(ConfigParams{
-            "MULTI:CPU,GPU",
```

Comment:
please update test case instead of removing test case

Reply:
Updated

Reply:
Updated